content/en/developers/dogstatsd/high_throughput.md

```diff
@@ -319,7 +319,32 @@ See [DataDog/datadog-go][1] for more information about the client configuration.
 {{% /tab %}}
 {{% tab "Java" %}}
 
-Telemetry will soon be added to the Java client.
+Starting with version `2.10.0` of the Java client.
+
+| Metric name                                      | Metric Type | Description                                                                                                                                                     |
+| ------------------------------------------------ | ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `datadog.dogstatsd.client.metrics`               | count       | Number of `metrics` sent to the DogStatsD client by your application (before sampling).                                                                         |
+| `datadog.dogstatsd.client.events`                | count       | Number of `events` sent to the DogStatsD client by your application.                                                                                            |
+| `datadog.dogstatsd.client.service_checks`        | count       | Number of `service_checks` sent to the DogStatsD client by your application.                                                                                    |
+| `datadog.dogstatsd.client.bytes_sent`            | count       | Number of bytes successfully sent to the Agent.                                                                                                                 |
+| `datadog.dogstatsd.client.bytes_dropped`         | count       | Number of bytes dropped by the DogStatsD client.                                                                                                                |
+| `datadog.dogstatsd.client.packets_sent`          | count       | Number of datagrams successfully sent to the Agent.                                                                                                             |
+| `datadog.dogstatsd.client.packets_dropped`       | count       | Number of datagrams dropped by the DogStatsD client.                                                                                                            |
+| `datadog.dogstatsd.client.packets_dropped_queue` | count       | Number of datagrams dropped because the DogStatsD client queue was full.                                                                                        |
+
+To disable telemetry, use the `enableTelemetry(false)` builder option:
```

Comment:
I would word this:

_Telemetry is enabled by default, to disable it use the `enableTelemetry(false)` builder option:_

Reply:
We already say it in the [common part of all client](https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?tab=java#client-side-telemetry): `Dogstatsd clients send telemetry metrics by default to the Agent.`.